src/ops/oracle.rs

```diff
@@ -208,13 +224,39 @@ impl Oracle {
                 commit_info.committed_txns.push(CommittedTxn {
                     ts,
                     conflict_keys: txn.conflict_keys.clone(),
+                    conflict_ranges: vec![],
                 })
             }
 
             (ts, false)
         }
     }
 
+    pub(crate) fn new_ingest_commit_ts(
```

Comment:
Where is the conflict check?

Reply:
https://github.com/wangnengjie/agatedb/blob/11161387d17c70bbe595afab67553ea1c9eef28e/src/ops/oracle.rs#L52

In `oracle::has_conflict`

Reply:
> https://github.com/wangnengjie/agatedb/blob/11161387d17c70bbe595afab67553ea1c9eef28e/src/ops/oracle.rs#L52
> 
> In `oracle::has_conflict`

But it is not called in `new_ingest_commit_ts`.

Reply:
Oracle's conflict check is only for read keys of update transaction.

Reply:
> Oracle's conflict check is only for read keys of update transaction. And ingest task doesn't have read keys.

Yes, that's right.

Reply:
> > Oracle's conflict check is only for read keys of update transaction. And ingest task doesn't have read keys.
> 
> Yes, that's right.

So we don't need to check conflict here during commiting ingest task~~